samples/sensor/max17262/src/main.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2020 Matija Tudan
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <devicetree.h>
+#include <drivers/sensor.h>
+#include <../drivers/sensor/max17262/max17262.h>
```

Comment:
This line looks like it doesn't belong here. You shouldn't be including a private header from a sensor driver.

Reply:
Done.